src/knot_cloud.c

```diff
@@ -297,23 +263,22 @@ static int create_cloud_queue(const char *id)
 
 	snprintf(queue_fog_name, sizeof(queue_fog_name), "%s-%s",
 		 MQ_QUEUE_FOG_OUT, id);
-
```

Comment:
Is removing this line necessary?

Reply:
Actually no, i think i removed by mistake

Reply:
done